rllib/evaluation/worker_set.py

```diff
@@ -397,6 +407,40 @@ def _from_existing(local_worker: RolloutWorker,
         workers._remote_workers = remote_workers or []
         return workers
 
+    def _get_dataset_and_shards(self, config: TrainerConfigDict,
+                                num_workers: int, local_worker: bool)\
+            -> (ray.data.dataset.Dataset,
+                List[ray.data.dataset.Dataset]):
+        assert config["input"] == "dataset"
+        assert "input_config" in config, (
+            "Must specify input_config dict if using Dataset input.")
+
+        input_config = config["input_config"]
```

Comment:
Just a thought:
In a follow-up PR, we should unify these config keys. Maybe something like:

```
Dataset:
  input_config: {
    type: "dataset",
    path: [...]
  }

JSON File:
  input_config:
    type: "JSONReader",
    path: [...]  <- filename or path w/ JSON files in it

Sampler (default):
  input_config:
    type: "sampler",
    env(??):  # <- could replace the "env" config key
```

Reply:
good idea. I added a big TODO near the input key.